packages/edit-post/src/components/header/fullscreen-mode-close/index.js

```diff
@@ -28,16 +39,30 @@ function FullscreenModeClose() {
 		return null;
 	}
 
+	let buttonIcon = <Icon size="36px" icon={ wordpress } />;
+
+	if ( siteIconURL ) {
+		buttonIcon = (
+			<img
+				className="edit-post-fullscreen-mode-close_site-icon"
+				src={ siteIconURL }
+				alt="site-icon"
+			/>
+		);
+	} else if ( isRequestingSiteIcon ) {
+		buttonIcon = null;
+	}
+
 	return (
 		<Button
-			className="edit-post-fullscreen-mode-close"
-			icon={ wordpress }
-			iconSize={ 36 }
+			className="edit-post-fullscreen-mode-close has-icon"
 			href={ addQueryArgs( 'edit.php', {
 				post_type: postType.slug,
 			} ) }
 			label={ get( postType, [ 'labels', 'view_items' ], __( 'Back' ) ) }
-		/>
```

Comment:
If  I remember properly since now, this is not an icon button, we might want to add a `showTooltip` prop to show the tooltip.

Reply:
Adding the `showTooltip` worked  👍